### DISCUSSION TITLE
Types for @paypal/checkout-server-sdk?

### DISCUSSION THREAD

**Discussion by chiyc** [DefinitelyTyped/DefinitelyTyped#57699] [Request a new @types package] [answered]
Are there types available for this PayPal SDK?

https://www.npmjs.com/package/@paypal/checkout-server-sdk
https://github.com/paypal/Checkout-NodeJS-SDK

I've submitted a technical support case to PayPal, and there are currently no public plans on their end to release type declarations for this package.

I've started working on my own declarations, but what I have is pretty minimal so far.

**Comment by martin-badin** [comment]
Hi, I started to work on the types but it is necessary to create types for the second package `@paypal/paypalhttp` as well.

https://github.com/martin-badin/DefinitelyTyped/tree/paypal__checkout-server-sdk/create/types/paypal__checkout-server-sdk

**Comment by martin-badin** [comment]
I created first PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57752

**Comment by martin-badin** [comment]
I created second PR #57791. That is still not final but I can create updates after merge.

**Reply by martin-badin** [reply, marked as answer]
All PR are merged
